pkgs/development/python-modules/pyslurm/default.nix

```diff
@@ -1,19 +1,27 @@
-{ lib, fetchFromGitHub, buildPythonPackage, cython, slurm }:
+{ lib, fetchFromGitHub, buildPythonPackage, cython, slurm, nose }:
 
 buildPythonPackage rec {
   pname = "pyslurm";
-  version = "20180427";
+  version = "20180604";
 
   src = fetchFromGitHub {
     repo = "pyslurm";
     owner = "PySlurm";
-    rev = "3900e1afac9ffd13c80c57d8c39933d42eb7bad7";
-    sha256 = "1a183ig4sdbc70rx2yyaslyq61wkbsf8cbim1jj0kzrp65nf0vls";
+    rev = "9dd4817e785fee138a9e29c3d71d2ea44898eedc";
+    sha256 = "14ivwc27sjnk0z0jpfgyy9bd91m2bhcz11lzp1kk9xn4495i7wvj";
   };
 
   buildInputs = [ cython slurm ];
   setupPyBuildFlags = [ "--slurm-lib=${slurm}/lib" "--slurm-inc=${slurm.dev}/include" ];
 
+  # Test cases need /etc/slurm/slurm.conf and require a working slurm installation
+  doCheck = false;
+
+  checkInputs = [ nose ];
+  checkPhase = ''
+    nosetests -v
+  '';
```

Comment:
When we're not running tests, do we prefer to leave the config showing how you would run tests if you had a system with all the pre-requisites setup, or do we prefer to delete it all to keep the default.nix lean?

Reply:
You could run such a test, for example, in `nixos/tests/slurm` (assuming that the test suite is included in the `pyslurm` installation).